### PR TITLE
Dots Update 2 + website appearance cleanup

### PR DESCRIPTION
**Pull request by ryanctan**
Visualization:
- Question mark drawn on top of dots; pop-ups look more like Explorer page
- Tooltips look better and size according to name length; bug fixed where they would persist after refreshing dots
- Visualization description appearance ready for link to Book
- When grouping and simultaneously coloring by gender, each group is sorted by gender
- Separate "Unknown" group when drawing based on some information on "fake" travelers (entries not representing specific individuals)
- Travel length resized such that travels <1 year are minimum dot size
- "Entry length" changed to "Word count"

Site Appearance:
- Fixed spacing issues on Search and Explore page

### CODE REVIEW

File: public/components/search-field/date-search.pug [hunk 1]
```diff
@@ -1,6 +1,5 @@
 .form-group
-    label {{title}}
-    
+
```
Comment (epicfaace) [inline public/components/search-field/date-search.pug]: Why did you remove the "Travel date" label?

**Review by epicfaace — APPROVED** [review]
Visualizations look good!

I only see the question mark when selecting "Color by" for the visualization -- is that intended?

File: public/visualization.component.ts [hunk 2]
```diff
@@ -128,161 +141,97 @@ export class VisualizationComponent {
...
+            let entriesInGroup = (entryGroups[i] as { entries: any[], request: any }).entries;
 
             d3.select('svg').append("text")
                 .attr("x", x)
                 .attr("y", y)
                 .text(function (d) { return group.title; });
             y += 15;
 
-            let dotGroup = this.drawDots(entriesInGroup, colorBy, sizeBy, y);
+            if (separateFakes) {
```
Comment (epicfaace) [inline public/visualization.component.ts:161]: It would be nice to extract this into a separate function later to make sure it's doing the right thing... but not a priority now.

File: public/components/search-field/date-search.pug (hunk 1 above)
Comment (ryanctan) [reply public/components/search-field/date-search.pug]: My mistake; it appears to be a redundant empty value on the Search page (deleting it removes the awkward space) but it wipes it out on the Explore page. I've restored it now.
